### PR TITLE
Add integration tile for Mendix

### PR DESCRIPTION
**Pull request by pappachino**
### What does this PR do?

The PR adds a tile for the datadog integration for the Mendix cloud platform.

### Motivation

[Mendix](https://mendix.com/) is a low-code application development platform.
The main purpose is to add visibility for the out-of-the-box Datadog integration for applications deployed to Mendix public cloud.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -85,6 +85,7 @@
 /lighthouse/                             @DataDog/agent-integrations
 /logstash/                               @ervansetiawan ervansetiawan@gmail.com
 /logzio/                                 @DataDog/agent-integrations
+/mendix/                                 @mendix/cloud @DataDog/agent-integrations
```
Comment (eho1307) [inline .github/CODEOWNERS:88]:
```suggestion
/mendix/                                 @mendix/cloud @DataDog/agent-integrations @DataDog/marketplace-review
```

File: mendix/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a market [leading low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
```
Comment (eho1307) [inline mendix/README.md:5]:
```suggestion
[Mendix][1] is a [low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
```

Removing marketing forward copy

File: mendix/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
+
+In addition to the native monitoring capabilities out of the box, Mendix also supports integrations to leading 3rd party application performance management vendors such as DataDog. Refer to our documentation to see all our [supported integrations][3].
```
Comment (eho1307) [inline mendix/README.md:9]:
```suggestion
```

File: mendix/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+The Mendix integration with DataDog enables you to consume and monitor your Mendix ecosystem. The [DataDog integration][4] from Mendix will enable you to monitor
+Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics and any custom metrics you choose to
+configure in Mendix Studio Pro.
```
Comment (eho1307) [inline mendix/README.md:13]:
```suggestion
[This integration][4] allows you to monitor your Mendix ecosystem, including Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics. Customize the metrics you send to Datadog in Mendix Studio Pro.
```

File: mendix/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+
+> **Note**:  This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
```
Comment (eho1307) [inline mendix/README.md:16]:
```suggestion
**Note**: This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
```

File: mendix/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Setup
+
+To enable the DataDog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
```
Comment (eho1307) [inline mendix/README.md:21]:
```suggestion
To enable the Datadog integration for Mendix applications running on Mendix cloud follow the instructions [here][6].
```

File: mendix/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Data Collected
+
+The list of metrics available for monitoring once the integration is enabled is detailed in the [Mendix documentation][7].
```
Comment (eho1307) [inline mendix/README.md:26]: If you're sending in metrics, you should include the metadata.csv file that lists the metrics being sent in.

For more info, refer [here](https://docs.datadoghq.com/developers/integrations/check_references/#metrics-metadata-file).

File: mendix/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+
+## Support
+
+Need help? Contact [Mendix support][5].
+
+[1]: https://mendix.com/
+[2]: https://www.mendix.com/blog/a-low-code-leader-composing-the-modern-enterprise-with-mendix/
+[3]: https://docs.mendix.com/developerportal/operate/monitoring-with-apm/
```
Comment (eho1307) [inline mendix/README.md:35]:
```suggestion
```

File: mendix/manifest.json [hunk 9]
```diff
@@ -0,0 +1,58 @@
...
+  "manifest_version": "2.0.0",
+  "app_uuid": "4119b134-c828-4e14-95b5-585bb13d314a",
+  "app_id": "mendix",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Market leading low-code IDE for application development",
```
Comment (eho1307) [inline mendix/manifest.json:11]:
```suggestion
    "description": "Monitor Mendix environment metrics",
```
This should be a description of the integration rather than Mendix.

File: mendix/manifest.json [hunk 10]
```diff
@@ -0,0 +1,58 @@
...
+    },
+    {
+      "media_type": "image",
+      "caption": "Cloud developer portal view of resource packs",
+      "image_url": "images/cloud_resource_pack_overview.png"
+    }   
+    ],
+    "classifier_tags": [
+      "Supported OS::Windows",
+      "Category::Cloud"
```
Comment (eho1307) [inline mendix/manifest.json:47]:
```suggestion
      "Supported OS::Windows",
      "Category::Cloud",
      "Category::Automation",
      "Offering::Integration"
```

File: mendix/manifest.json [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    ]
+  },
+  "assets": {},
```
Comment (eho1307) [inline mendix/manifest.json:50]: We require integrations to include an out-of-the-box dashboard. Please create one in your sandbox account and export it as a JSON file. The file can be attached to this PR. This ensures customers don't need to set up their own dashboards and receive immediate value after installing the integration. Refer [here](https://datadoghq.dev/integrations-core/guidelines/dashboards/#best-practices) for more info. 

Note: Include a screenshot of your dashboard as an image

File: mendix/manifest.json [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Market leading low-code IDE for application development",
+    "title": "Mendix",
+    "media": [
+    {
+      "media_type": "image",
+      "caption": "Mendix Studio Pro 9",
+      "image_url": "images/mx9_splash_screen.png"
+    },
```
Comment (eho1307) [inline mendix/manifest.json:18]:
```suggestion
```

Please only include images that are relevant to the integration.

File: mendix/manifest.json [hunk 13]
```diff
@@ -0,0 +1,58 @@
...
+    {
+      "media_type": "image",
+      "caption": "Domain model with Data Hub",
+      "image_url": "images/domain-model-with-data-hub.png"
+    },
+    {
+      "media_type": "image",
+      "caption": "Cloud developer portal view of available plans",
+      "image_url": "images/cp_environments_popup.png"
+    },
```
Comment (eho1307) [inline mendix/manifest.json:33]:
```suggestion
```

Please only include images that are relevant to the integration.

**Review by eho1307 — CHANGES_REQUESTED** [review]
Thank you for opening this PR! I've added comments inline. Feel free to reach out with any questions!

File: mendix/README.md (hunk 2 above)
Comment (pappachino) [reply mendix/README.md]: I've fixed the text

File: mendix/README.md (hunk 7 above)
Comment (eho1307) [reply mendix/README.md]: @pappachino Unfortunately, there isn't a way to export a list of metrics with their descriptions.

File: mendix/README.md [hunk 14]
```diff
@@ -0,0 +1,33 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a [low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
```
Comment (alai97) [inline mendix/README.md:5]:
```suggestion
[Mendix][1] is a [low-code IDE][2] and visual model-driven development environment that developers can use to build apps on the Mendix Platform.
```

File: mendix/README.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+
+With Mendix, you can easily create, change, integrate, test and deploy your applications. Moreover, you can manage branch lines and security as well as extend your app with custom code using the built-in editors.
```
Comment (alai97) [inline mendix/README.md:7]:
```suggestion
With Mendix, you can create, change, integrate, test, and deploy your applications. You can manage branch lines and security as well as extend your app with custom code by using the built-in editor.
```

File: mendix/README.md [hunk 16]
```diff
@@ -0,0 +1,33 @@
...
+
+[This integration][3] allows you to monitor your Mendix ecosystem, including Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics. Customize the metrics you send to Datadog in Mendix Studio Pro.
```
Comment (alai97) [inline mendix/README.md:9]:
```suggestion
[The Mendix integration][3] allows you to monitor your Mendix ecosystem, including Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, and SaaS (Software as a Service) environment metrics. Customize the metrics you send to Datadog in Mendix Studio Pro.
```

File: mendix/README.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+
+**Note**: This integration is applicable to Mendix Cloud & Mendix Cloud Dedicated deployment models.
```
Comment (alai97) [inline mendix/README.md:11]:
```suggestion
**Note**: This integration is applicable to Mendix Cloud and Mendix Cloud Dedicated deployment models.
```

File: mendix/README.md [hunk 18]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Setup
+
+To enable the Datadog integration for Mendix applications running on Mendix cloud follow the instructions [here][5].
```
Comment (alai97) [inline mendix/README.md:16]:
```suggestion
To enable the Datadog integration for Mendix applications running on Mendix Cloud, see the [Mendix setup documentation][5].
```

File: mendix/README.md [hunk 19]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Data Collected
+
+The list of metrics available for monitoring once the integration is enabled is detailed in the [Mendix documentation][6].
+
```
Comment (alai97) [inline mendix/README.md:22]:
```suggestion

### Metrics

For a list of metrics available once the integration is enabled, see the [official Mendix documentation][6].

### Events

The Mendix integration does not include any events.

### Service Checks

The Mendix integration does not include any service checks.

```

File: mendix/README.md [hunk 20]
```diff
@@ -0,0 +1,33 @@
...
+
+## Support
```
Comment (alai97) [inline mendix/README.md:24]:
```suggestion
## Troubleshooting
```

**Review by alai97 — APPROVED** [review]
Some copy suggestions—looks good pending @eho1307's feedback!

File: mendix/README.md (hunk 8 above)
Comment (Sakshi-Dhakad) [reply mendix/README.md]: @EH - Is there a reason to remove this link? We would prefer to keep it as it provides an overview of how the integration works with Mendix overall

Comment (Sakshi-Dhakad) [reply mendix/README.md]: cc - @pappachino

File: mendix/README.md (hunk 7 above)
Comment (eho1307) [reply mendix/README.md]: If you include a list of metrics, we display it on our documentation site so customers can see a list of them on the same page and know what to expect. See an example under the 'Data Collected' section [here](https://docs.datadoghq.com/integrations/1e/).

File: mendix/README.md (hunk 3 above)
Comment (eho1307) [reply mendix/README.md]: This is not necessary since customers looking at this information already know an integration exists. Please remove this or reword it to 'Refer to [our documentation][3] for more information about monitoring Mendix with APM tools'.

File: mendix/README.md (hunk 2 above)
Comment (eho1307) [inline mendix/README.md]:
```suggestion
[Mendix][1] is a [low-code IDE][2] for professional developers. A powerful visual model-driven development environment to build apps on the Mendix Platform.
```

Comment (eho1307) [reply mendix/README.md]: Please remove marketing forward language.

File: mendix/README.md (hunk 4 above)
Comment (eho1307) [reply mendix/README.md]: Please add in this change for clarity. There's a lot of repetition in the previous version.

File: mendix/README.md (hunk 6 above)
Comment (eho1307) [reply mendix/README.md]: Please address this comment. Datadog should be spelt with only one uppercase D rather than two.

File: mendix/manifest.json (hunk 9 above)
Comment (eho1307) [reply mendix/manifest.json]: Please remove marketing forward language

File: mendix/manifest.json (hunk 10 above)
Comment (eho1307) [reply mendix/manifest.json]: Please address this comment

File: mendix/CHANGELOG.md [hunk 21]
```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Mendix
+
+## 1.0.0
+* [Added] Initial revision of Mendix integration
```
[the remark this answers is not in the document]
Comment (eho1307) [reply mendix/CHANGELOG.md]: Please address this comment

**Review by eho1307 — CHANGES_REQUESTED** [review]

File: mendix/README.md (hunk 2 above)
Comment (eho1307) [reply mendix/README.md]: Apologies, ignore this comment. I was looking at an old version of the PR.

**Review by iliakur — APPROVED** [review]

File: mendix/README.md (hunk 8 above)
Comment (pappachino) [reply mendix/README.md:44]: @eho1307 We would like to keep the referenced link as it provides a generic starting point for all APMs including Datadog. I've updated the README accordingly.

File: mendix/README.md (hunk 20 above)
Comment (pappachino) [reply mendix/README.md]: Fixed

File: mendix/README.md (hunk 19 above)
Comment (pappachino) [reply mendix/README.md]: Thanks, I've fixed it

File: mendix/README.md (hunk 16 above)
Comment (pappachino) [reply mendix/README.md]: Thanks, I've fixed it.

File: mendix/README.md (hunk 15 above)
Comment (pappachino) [reply mendix/README.md]: Thanks, I've fixed it.

File: mendix/manifest.json (hunk 12 above)
Comment (pappachino) [reply mendix/manifest.json:18]: As we don't have any dashboards yet, I guess this is fine for the first pass. We could revisit this once we have screenshots of relevant DD dashboards

File: mendix/manifest.json (hunk 13 above)
Comment (pappachino) [reply mendix/manifest.json:33]: As we don't have any dashboards yet, I guess this is fine for the first pass. We could revisit this once we have screenshots of relevant DD dashboards

File: mendix/manifest.json (hunk 11 above)
Comment (pappachino) [reply mendix/manifest.json]: @eho1307 Thanks for the pointer. 
But currently, we don't have a starter template the customers could use. We would need to give it some thought before we can make it available.

So for the first pass could we publish the integration without any out-of-the-box dashboard ?

File: mendix/README.md (hunk 7 above)
Comment (pappachino) [reply mendix/README.md]: Thanks, @eho1307. I've dropped a metadata.csv file with all the generic metrics available from the Mendix integration.
We won't be able to list all of them, as the metric availability might vary from Mendix app to app.

**Comment by pappachino** [comment]
@eho1307 @alai97  I've addressed most of the comments as suggested. Do let me know if you have any further comments and if the PR can be accepted. Thanks for reviewing the PR

File: mendix/README.md [hunk 22]
```diff
@@ -0,0 +1,47 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a [low-code IDE][2] and a visual model-driven development
+environment that developers can use to build apps on the Mendix Platform.
```
Comment (eho1307) [inline mendix/README.md:6]:
```suggestion
[Mendix][1] is a [low-code IDE][2] and a visual model-driven development environment that developers can use to build apps on the Mendix Platform.
```

File: mendix/README.md [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+
+With Mendix, you can easily create, change, integrate, test, and deploy your
+applications. You can manage branch lines and security as well as extend your
+app with custom code by using the built-in editors.
```
Comment (eho1307) [inline mendix/README.md:10]:
```suggestion
With Mendix, you can easily create, change, integrate, test, and deploy your applications. You can manage branch lines and security as well as extend your app with custom code by using the built-in editors.
```

File: mendix/manifest.json (hunk 12 above)
Comment (eho1307) [reply mendix/manifest.json:18]: Please remove this image

File: mendix/manifest.json (hunk 13 above)
Comment (eho1307) [reply mendix/manifest.json:33]: Please remove this image

**Review by eho1307 — CHANGES_REQUESTED** [review]

File: mendix/manifest.json (hunk 12 above)
Comment (pappachino) [reply mendix/manifest.json]: I've removed the image.

File: mendix/README.md [hunk 24]
```diff
@@ -0,0 +1,43 @@
+# Mendix
+
+## Overview
+
+[Mendix][1] is a [low-code IDE][2] and a visual model-driven development environment that developers can use to build apps on the Mendix Platform.
+
+With Mendix, you can easily create, change, integrate, test, and deploy your applications. You can manage branch lines and security as well as extend your app with custom code by using the built-in editors.
+
+[The Mendix integration][3] allows you to monitor your Mendix ecosystem, including Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, SaaS(Software as a Service) environment metrics. Customize the metrics you send to Datadog in Mendix Studio Pro.
```
Comment (emarsha94) [inline mendix/README.md:9]:
```suggestion
[Mendix][1] is a [low-code IDE][2] and a visual model-driven development environment that developers can use to build apps on the Mendix Platform. With Mendix, you can easily create, change, integrate, test, and deploy your applications. You can manage branch lines and security as well as extend your app with custom code by using the built-in editors.

[The Mendix integration][3] allows you to monitor your Mendix ecosystem, including Mendix Runtime metrics, Java Virtual Machine (JVM) metrics, database, and SaaS(Software as a Service) environment metrics. You can customize the metrics you send to Datadog in Mendix Studio Pro.
```

Comment (emarsha94) [reply mendix/README.md:9]: Can you provide insight on the value of monitoring the metrics?

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Could you provide your logo in svg format? The light mode logo is cut off. Thanks!

**Comment by emarsha94** [comment]
> Hi @pappachino The only remaining comment from my end is regarding the dashboard. Once that's been added in, your PR is good to go from my end!

Hi @pappachino - Could you add a dashboard to the integration? Thanks!

**Comment by pappachino** [comment]
> > Hi @pappachino The only remaining comment from my end is regarding the dashboard. Once that's been added in, your PR is good to go from my end!
> 
> Hi @pappachino - Could you add a dashboard to the integration? Thanks!

@emarsha94  we'll be adding a dashboard to the PR next week. Also, I'll provide svg format logo. 
Thanks for the review.

**Comment by pappachino** [comment]
> Could you provide your logo in svg format? The light mode logo is cut off. Thanks!

The logo was already in svg format. Can you check again. Thanks!

**Comment by pappachino** [comment]
> Hi @pappachino The only remaining comment from my end is regarding the dashboard. Once that's been added in, your PR is good to go from my end!

I've added a dashboard. Let me know if there is anything else, thanks!

File: mendix/manifest.json (hunk 11 above)
Comment (pappachino) [reply mendix/manifest.json]: I've added a dashboard and a sample image.

**Review by eho1307 — APPROVED** [review]

**Comment by pappachino** [comment]
@emarsha94  I've addressed the review comments. 
Do let me know if you have any further comments and if the PR can be accepted.

File: mendix/manifest.json [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+        "media_type": "image",
+        "caption": "Mendix Application Overview dashboard",
+        "image_url": "images/mendix_dashboard.png"
+    }
+    ],
+    "classifier_tags": [
+      "Supported OS::Windows",
+      "Category::Cloud",
+      "Category::Automation",
+      "Offering::Integration"
```
Comment (emarsha94) [inline mendix/manifest.json:44]:
```suggestion
    "classifier_tags": [
      "Supported OS::Windows",
      "Category::Cloud",
      "Category::Automation",
      "Offering::Integration",
      “Submitted Data Type::Metrics”
```

**Review by emarsha94 — CHANGES_REQUESTED** [review]
Added 2 more suggestions. Once those are added, this looks good to me :) Thanks!

**Comment by pappachino** [comment]
> Added 2 more suggestions. Once those are added, this looks good to me :) Thanks!

I've addressed the suggestions.

File: mendix/README.md [hunk 26]
```diff
@@ -0,0 +1,43 @@
...
+
+**Note**: This integration is applicable to Mendix Cloud and Mendix Cloud Dedicated deployment models.
+
+
```
Comment (emarsha94) [inline mendix/README.md:13]:
```suggestion
```

File: mendix/CHANGELOG.md [hunk 27]
```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Mendix
+
+## 1.0.0 / 2023-03-27
```
Comment (emarsha94) [inline mendix/CHANGELOG.md:3]:
```suggestion
## 1.0.0 / 2023-03-31
```

**Review by emarsha94 — APPROVED** [review]
Looks good to me! Two minor changes requested. Thanks!
